src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/FileBinding.java

```diff
@@ -97,6 +98,7 @@ protected void copy(FilePath secret, FileCredentials credentials) throws IOExcep
         secret.copyFrom(credentials.getContent());
     }
 
+    @Symbol("file")
```

Comment:
Namespace collision with that? Symbols are scoped per extension point. What other `MultiBinding` would want to be called `file`?